File: Python/BinarytoDecimal.py
#binary to decimal

a=input('enter a binary code: ')
a=a[::-1]
print('Binary code:',a)
b=[]
for x in a:
    b=b+[int(x)]
n=0
s=0
for i in b:
    if i==1:
        s+=i*(2**n)
    n+=1
print('Decimal conversion:',s)

File: Go/Binary-to-decimal .go
package main  
  
import (  
 "fmt"  
 "strconv"  
)  
  
func main() {  
 var binary string  
 fmt.Print("Enter Binary Number:")  
 fmt.Scanln(&binary)  
 output, err := strconv.ParseInt(binary, 2, 64)  
 if err != nil {  
  fmt.Println(err)  
  return  
 }  
  
 fmt.Printf("Output %d", output)  
  
}  
